src/dkpy/structured_singular_value.py

```diff
@@ -319,58 +321,60 @@ def _ssv_at_omega(
         return mu, D_scales, info
 
 
-def _variable_from_block_structure(block_structure: np.ndarray) -> cvxpy.Variable:
```

Comment:
Might make sense to move this into the uncertainty block object. Then you can put the `NotImplementedError`s into the subclasses when calling this